hack/test-cnv.sh

```diff
@@ -89,4 +89,5 @@ ${TESTS_BINARY} \
     -kubectl-path="$(which oc)" \
     -utility-container-prefix=quay.io/kubevirt \
     -test.timeout=2h \
+    -ginkgo.flakeAttempts=3 \
```

Comment:
can you please use `--ginkgo.flake-attempts`?
see https://onsi.github.io/ginkgo/MIGRATING_TO_V2#changed-command-line-flags

Reply:
updated.

Reply:
sorry, bad idea: we are still running with old ginkgo